www/src/pages/contributing/events.js

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Helmet } from "react-helmet"
+
+import Layout from "../../components/layout"
+import { itemListContributing } from "../../utils/sidebar/item-list"
+import Container from "../../components/container"
+import EmailCaptureForm from "../../components/email-capture-form"
+import DocSearchContent from "../../components/docsearch-content"
+import Events from "../../components/events/events"
+
+const IndexRoute = props => (
+  <Layout location={props.location} itemList={itemListContributing}>
+    <DocSearchContent>
+      <Container>
+        <Helmet>
+          <title>Community Events</title>
+        </Helmet>
+        <h1 id="contributing-gatsby" css={{ marginTop: 0 }}>
+          Gatsby Community Events
+        </h1>
+        <p>
```

Comment:
Should we link to the new community events page? https://www.gatsbyjs.org/contributing/organize-a-gatsby-event/

Reply:
@lindaleebumblebee what do you think here?

Reply:
Not sure what Linda votes, but I vote yes! "Or, organize your own event!"